roles/test_operator/defaults/main.yml

```diff
@@ -49,7 +59,7 @@ cifmw_test_operator_tempest_config:
         {{ cifmw_test_operator_tempest_exclude_list | default('') }}
       concurrency: "{{ cifmw_test_operator_concurrency }}"
       externalPlugin: "{{ cifmw_test_operator_tempest_external_plugin | default([]) }}"
-    tempestconfRun: "{{ cifmw_tempest_tempestconf_config | default(omit) }}"
+    tempestconfRun: "{{ cifmw_tempest_tempestconf_config_defaults | combine(cifmw_tempest_tempestconf_config) }}"
```

Comment:
I've just tested this quickly locally. This will fail because of `cifmw_tempest_tempestconf_config` is not defined. 

```
...
'cifmw_tempest_tempestconf_config' is undefined\n\nThe error appears to be in '/home/cloud-user/ci-framework/roles/test_operator/tasks/main.yml': line 117, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Start Tempest tests\n 
```

Reply:
The `cifmw_tempest_tempestconf_config_defaults` variable should probably look like this:
```
cifmw_tempest_tempestconf_config_defaults:
  overrides: |
    enforce_scope.nova true
    enforce_scope.neutron true
    enforce_scope.glance true
    enforce_scope.cinder true
    enforce_scope.keystone true
    enforce_scope.placement true
```

Reply:
@kopecmartin as you pointed out this seems to be working together with the change of `cifmw_tempest_tempestconf_config_defaults` (above ^^):

```
tempestconfRun: "{{ cifmw_tempest_tempestconf_config_defaults | combine(cifmw_tempest_tempestconf_config | default({})) }}"
```

Reply:
@kopecmartin as I'm thinking about this. The `combine` will does not work then, right? Because `override` section is a string.

Reply:
that's a problem, yes, does it have to be a string?

Reply:
.. as overrides are a string, we need to set these defaults in another way and leave 'overrides' for jobs to set. 
We have 2 options then .. try to use 'deployerInput' or set srbac by default in python-tempestconf. I'd like to avoid the latter.